doc/classes/Vector3.xml

```diff
@@ -404,11 +404,29 @@
 			Down unit vector.
 		</constant>
 		<constant name="FORWARD" value="Vector3(0, 0, -1)">
-			Forward unit vector. Represents the local direction of forward, and the global direction of north.
+			Forward unit vector. Represents the local direction of forward, and the global direction of north. Keep in mind that, in Godot, the forward vector is generally opposed to where imported 3D assets are facing. Use [constant Vector3.MODEL_FRONT] and similar constants to manipulate transforms in imported 3D asset space.
```

Comment:
```suggestion
			Forward unit vector. Represents the local direction of forward, and the global direction of north. Keep in mind that the forward direction for lights, cameras, etc is different from 3D assets like characters, which face towards the camera by convention. Use [constant Vector3.MODEL_FRONT] and similar constants when working in 3D asset space.
```

Reply:
I integrated this change in https://github.com/godotengine/godot/pull/76082.